docs/_includes/attr-doc.adoc

```diff
@@ -65,16 +68,27 @@ You can also set the base path to images (default: _empty_), icons (default: `./
 
 When you find yourself typing the same text repeatedly, or text that often needs to be updated, consider assigning it to a document attribute and use that instead.
 
+NOTE: If you're used to XML, you'll know document attributes as User Entities.
+
 Attribute entries have the following characteristics:
 
-* can be assigned to a document via the CLI or in the document's header and body
-* have default values
-** their default values are set when the user leaves the value field empty
-* some have alternate values
-* they can be unset (turned off) with a `!`
-* they override config files, intrinsic attributes
-* they do *not* override attributes issued from the command line
+Attributes entries can::
+* be assigned to a document:
+** through the CLI
+** in the document's header
+** in the document's body
+* be unset (turned off) with a `!`
+* have default values, which are set when the user leaves the value field empty
+* have alternate values
+* span multiple lines
+* override config files, intrinsic attributes
```

Comment:
Let's change this to "override some intrinsic attributes"

since not all intrinsic attributes can be overridden and we don't have config files.